sorc/orog_mask_tools.fd/orog.fd/mtnlm7_oclsm.f

```diff
@@ -163,8 +163,8 @@
       SUBROUTINE TERSUB(IMN,JMN,IM,JM,NM,NR,NF0,NF1,NW,EFAC,BLAT,
      &     OUTGRID,INPUTOROG)
 !jaa      use ipfort
+      use machine
       implicit none
-      include 'machine.h'
```

Comment:
OK, we could have gotten doxygen to work with the .h file, but I'm also fine with this change, since it makes more sense. Why have fortran code in a .h file? That makes so sense...